browser-test/src/applicant/questions/dropdown.test.ts

```diff
@@ -59,7 +59,7 @@ test.describe('Dropdown question for applicant flow', () => {
         },
         /* clickSubmit= */ false,
       )
-      await adminQuestions.expectPreviewOptions(['Sample question option'])
+      await adminQuestions.expectPreviewOptions(['Sample question option\n'])
```

Comment:
We get an extra `\n` at the end of each line thanks to the text formatter, but it doesn't affect how the options render.